subsys/power/device.c

```diff
@@ -57,7 +57,7 @@ int sys_pm_suspend_devices(void)
 						DEVICE_PM_SUSPEND_STATE,
 						NULL, NULL);
 		if (device_retval[i]) {
-			LOG_ERR("%s suspend operation failed\n",
+			LOG_DBG("%s suspend operation failed\n",
```

Comment:
@vanti Thanks for the review,  my commit message describe the reason ```
Current PM policy allow devices
make the decision if going to sleep/deep sleep state, so it's not error
message if failed suspend devices, change it to debug level.
``` 
and if using error message, it will confuse end users who are not familiar zephyr power management.

Reply:
In this case the message is too strong. When I see "failed" it sounds like an error message. How about "%s did not enter suspend state"?

Reply:
@vanti updated.